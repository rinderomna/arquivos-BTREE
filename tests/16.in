8 tipo2 binario16.bin indice16.bin 3
1 id 809
1 cidade "SANTA TEREZINHA DO ITAIPU"
1 sigla NULO
1 cidade NULO
1 ano 1996
2 ano 1997 qtt 14
