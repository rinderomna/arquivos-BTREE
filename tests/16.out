19316.310000
1379.210000
